src/Components/FilmCard.css

```diff
@@ -0,0 +1,117 @@
+.film-card{
```

Comment:
`.film-card{` -> `.film-card {`

Reply:
I allways miss that, hope it will be last time :)